Sonja/update mod dash 2

Sonjajo opened the pull request:
Updated the moderator dashboard to have scrolling and added a few buttons. Redid the layout for the orgDetail that pops up to match figma. 
Still to be done:
- fix styles of left AppBar with the treasury version that Elizabeth and I talked about: https://mui-treasury.com/styles/tabs/#Apple
- add drawer for Notepad button
- add popup window for rejection history
- look into custom drawer left scrollbar, material UI doesn't have functionality we need

_Required for visual changes_

![image](https://user-images.githubusercontent.com/30053749/98892080-61b47000-2454-11eb-872d-00e66f157547.png)

CC: @kwfk

Sonjajo committed:
feat: add drawer and sticky app bar, some bugs in formatting still

Sonjajo committed:
refactor: trying to fix App Bar nesting

Sonjajo committed:
feat: add sticky to moderator

Sonjajo committed:
feat: more scrolling debugging

Sonjajo committed:
feat: woooo added scrollability with sticky headers and footers

elixawu commented:
minor change, but the accept and decline buttons on the bottom should be swapped order (accept on right)

kwfk commented on pages/mod2.tsx:
```diff
@@ -0,0 +1,231 @@
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import InboxIcon from '@material-ui/icons/MoveToInbox';
+import MailIcon from '@material-ui/icons/Mail';
```
collapse to one line import
i.e.
```typescript
import { Drawer, List } from '@material-ui/core'
```

kwfk commented on pages/mod2.tsx:
```diff
@@ -0,0 +1,231 @@
+  createStyles,
+} from '@material-ui/core/styles';
+import Drawer from '@material-ui/core/Drawer';
+import CssBaseline from '@material-ui/core/CssBaseline';
```
this is imported at base root so don't need to be imported here

kwfk commented on styles/Moderator.module.css:
```diff
@@ -1,47 +1,60 @@
 }
+
 .rightCol {
+  width: 70%;
```
if using a percentage, you want the elements on the same level to equal 100%

kwfk replied:
then changing this as `calc(100% - 400px)`

kwfk commented on styles/Moderator.module.css:
```diff
@@ -1,47 +1,60 @@
-  margin-top: 8px;
-  margin-left: 15px;
-  margin-right: 15px;
+  width: 400px;
```
if using pixels, you want the other element in the same level to span the rest of the width

kwfk replied:
i think leaving this as a pixel width would be better

kwfk commented on pages/registration.tsx:
```diff
@@ -1,5 +1,5 @@
 import { useFormik } from 'formik';
-import { Form } from 'interfaces';
+import { Form } from 'interfaces/index';
```
was this change necessary?

Sonjajo replied:
nope you're right, i reverted it

kwfk commented on components/moderator/OrgDetail/index.tsx:
```diff
@@ -1,42 +1,117 @@
+          <div className={styles.small}>Orientation</div>
+          <div className={styles.chips}>
+            {items.orientation.map((item) => (
+              // TODO: Add accessibility support
+              // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
```
these eslint-disables are actually not needed!

kwfk reviewed (changes requested):
nice work! it looks really nice. there are some minor changes i commented

you may want to add spacing between the demographic chips

Sonjajo committed:
feat: so i can switch branches

Sonjajo committed:
refactor: fixing merge conflict so i can merge with main

Sonjajo committed:
feat: add sample data that matches joi schema for backend linking

claalmve committed:
Merge branch 'main' into sonja/link-mod-backend

claalmve committed:
feat(changing types for org): changing types for org

n

claalmve committed:
feat(updating sample data): updating sample data

claalmve committed:
feat(updating moderator dash): updating moderator dash

Sonjajo committed:
refactor: refactor Moderator and components to work with prisma type

Sonjajo committed:
refactor: functional (but still ugly-ish) commit for GM demos

Sonjajo committed:
refactor: delete mod2.tsx since it was for testing only

kwfk committed:
fix(orgs page): remove org list page

Removed org list page because it now breaks the build after changing structure of sample data.

kwfk commented on interfaces/index.ts:
```diff
@@ -35,6 +35,27 @@ export type Form = {
   orientation: string[];
 };
 
+export type SchemaForJoi = {
+  applicationStatus: string;
+  organizationName: string;
+  contactName: string;
+  contactEmail: string;
+  organizationType: string;
+  workType: string;
+  address: string;
+  lat: number;
+  long: number;
+  missionStatement: string;
+  shortHistory: string;
+  keyValue: string;
+  lgbtqDemographic: string;
+  raceDemographic: string;
+  ageDemographic: string;
+  capacity: number;
+  ein: number;
+  foundingDate: number;
+};
```
is this being used?

kwfk commented on pages/moderator.tsx:
```diff
@@ -1,57 +1,205 @@
 
-export const getStaticProps: GetStaticProps = async () => {
-  const items: OrgApp[] = sampleOrgAppData;
+// export const getStaticProps: GetStaticProps = async () => {
+//   const items: OrgApp[] = sampleOrgAppData;
+//   return { props: { items } };
+// };
```
let's remove this

kwfk commented on pages/moderator.tsx:
```diff
@@ -1,57 +1,205 @@
+//   return { props: { items } };
+// };
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
```
remove unused prop `context`

kwfk commented on pages/moderator.tsx:
```diff
@@ -1,57 +1,205 @@
+    setOpenRight(false);
+  };
+
+  console.log(items);
```
remove console log

kwfk reviewed (changes requested):
looking good! just some minor clean up

Sonjajo committed:
refactor: address minor changes requested

Sonjajo committed:
Merge branch 'sonja/update-mod-dash-2' of https://github.com/calblueprint/nbjc into sonja/update-mod-dash-2

kwfk committed:
Merge branch 'main' into sonja/update-mod-dash-2

kwfk reviewed (approved):
Nice work!

kwfk committed:
fix(moderator): update filter and height calculation
